ftest.sh

```diff
@@ -312,7 +312,9 @@ if ${SETUP_ONLY:-false}; then
 fi
 
 # now run it!
-export PYTHONPATH=./util:../../utils/py/:./util/apricot:./install/lib/python2.7/site-packages:./install/lib/python3/site-packages
+export PYTHONPATH=./util:../../utils/py/:./util/apricot:\
+                  ../../../install/lib/python2.7/site-packages:\
+                  ../../../install/lib/python3/site-packages
```

Comment:
I'm not sure you can break long `PYTHONPATH` lines like this and have all of that leading whitespace at the front of the subsequent lines.  Either remove the whitespace or just add to `PYTHONPATH` in the subsequent lines.

Reply:
removed the space, hope that will work.